file.c

```diff
@@ -180,6 +201,15 @@ static int _rf_file_write_int8_complex(void *private, int16_t *iq_data, size_t s
 	return(HACKTV_OK);
 }
 
+static int _rf_file_write_uint16_complex(void *private, int16_t *iq_data, size_t samples)
+{
+	rf_file_t *rf = private;
+
+	fwrite(iq_data + INT16_MAX, sizeof(int16_t) * 2, samples, rf->f);
```

Comment:
This will write bad data, or crash. See `_rf_file_write_uint8_complex()` for an idea of how it should look.

Reply:
Done too.